File: NetCore/CascadingSelectBoxesSample/Models/State.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CascadingSelectBoxesSample.Models {
    public class State
    {
        public int ID { get; set; }
        public string Name { get; set; }

        public static State[] States = new State[] {
            new State() {
                ID = 1,
                Name = "Alabama"
            },
            new State() {
                ID = 2,
                Name = "Alaska"
            },
            new State() {
                ID = 3,
                Name = "Arizona"
            },
            new State() {
                ID = 4,
                Name = "Arkansas"
            },
            new State() {
                ID = 5,
                Name = "California"
            }
        };
    }
}

File: VB/NetCore/CascadingSelectBoxesSample/Models/State.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace CascadingSelectBoxesSample.Models
	Public Class State
		Public Property ID() As Integer
		Public Property Name() As String

		Public Shared States() As State = {
			New State() With {
				.ID = 1,
				.Name = "Alabama"
			},
			New State() With {
				.ID = 2,
				.Name = "Alaska"
			},
			New State() With {
				.ID = 3,
				.Name = "Arizona"
			},
			New State() With {
				.ID = 4,
				.Name = "Arkansas"
			},
			New State() With {
				.ID = 5,
				.Name = "California"
			}
		}
	End Class
End Namespace
